### PR TITLE
Migrate some liquid templates to use va-accordion

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

Now that we have Web Components available in `vets-website`, this PR is the start of migrating away from the accordions which rely on the `usa-accordion` class.

There are many more `usa-accordion`s left in the liquid templates (and in application code), but I've kept this PR to a fairly manageable size.

`<va-accordion>` documentation can be found [here](https://design.va.gov/storybook/?path=/docs/components-va-accordion--default), which may be helpful in the review.

## Testing done

Local browser testing

## Screenshots

An example of the changes to `merger-social-sco.html`:

image

An example of the changes made to `landing_page` and `administration-hub-rail`:

image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by kelsonic** [comment]
What's the problem we're using web-components to solve? My fear of moving everything to web components is if we were to ever move to a different templating engine (e.g. Gatsby) then we'd have to look up the source of each web component to make the transition, making it that much more difficult.

**Comment by bkjohnson** [comment]
The problem we're using web components to solve is having to maintain multiple sources of truth for our design system components ([see more here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/design-docs/2021-01-13-web-component-library.md#objective)).

More specifically, this initiative was started [as a direct response to this Slack thread](https://dsva.slack.com/archives/C52CL1PKQ/p1604587276472700), where the public websites team expressed frustration about not being able to use a `<Table>` component since it was React-specific. This is why we want to start the migration with your team.

---

> My fear of moving everything to web components is if we were to ever move to a different templating engine (e.g. Gatsby) then we'd have to look up the source of each web component to make the transition, making it that much more difficult.

Can you elaborate on this? I don't understand why you would need to look at a web component's source. For example, a `<va-accordion>` in HTML/liquid is the same thing as `<va-accordion>` in JSX.

**Comment by kelsonic** [comment]
> Can you elaborate on this? I don't understand why you would need to look at a web component's source. For example, a <va-accordion> in HTML/liquid is the same thing as <va-accordion> in JSX.

Oh awesome, I wasn't aware of that 🙂

**Review by alexandec — APPROVED** [review]

File: src/site/components/navigation-sidebar.html [hunk 1]
```diff
@@ -30,79 +30,6 @@
         <div class="left-side-nav-title {% if newGrid %}vads-u-padding-x--0{% endif %}"><i class="icon-small white hub-icon-{{ relatedPages.metadata.hub }} hub-background-{{ relatedPages.metadata.hub }}"></i><h4>{{ relatedPages.metadata.name }}</h4></div>
       {% endif %}
 
-      {% if spoke != empty %}
```
[the remark this answers is not in the document]
Comment (ncksllvn) [reply src/site/components/navigation-sidebar.html:33]: Pretty sure this whole thing is tech debt

**Review by ncksllvn — APPROVED** [review]

**Pull request merged**